src/parsers/pulldown.rs

```diff
@@ -204,23 +215,25 @@ impl CommonMarkViewerInternal {
         options: &CommonMarkOptions,
         text: &str,
         populate_split_points: bool,
-    ) -> egui::InnerResponse<()> {
+    ) -> (egui::InnerResponse<()>, Vec<CheckboxClickEvent>) {
```

Comment:
It returns a Vec, but usually there is only one thing that the user clicks.
... Although there is multitouch and whatnot, so I think this is a safe option.